### DISCUSSION TITLE
Alpine script error

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [alpinejs/alpine#2146] [1. Help] [answered]
I came across this blog [post](https://ryangjchandler.co.uk/posts/pseudo-computed-properties-in-alpine) and I wanted to recreate the example myself, but when I run my code in jsfiddle, it just prompt me "Script error." That is not very helpful. Here is my code:
https://jsfiddle.net/wah3qpoy/

**Comment by Tim-Wils** [comment, marked as answer]
@Chandler-Zhu The script is not deferred in jsFiddle, that's why you get the error. I removed it from the sources and added it to the html just like how alpinejs.dev does it https://jsfiddle.net/hn1wuLe5/
